src/components/Calendar/src/Calendar.vue

```diff
@@ -6,12 +6,11 @@
 	>
 		<div :class="$s.CalendarHeader">
 			<m-button
-				:disabled="shouldDisableHeaderButton(-1)"
+				:disabled="isCalendarNavDisabled(-1)"
 				color="#f2f2f2"
 				size="small"
 				variant="primary"
-				tabindex="-1"
```

Comment:
Removed `tabindex` for accessibility.

The `tabindex` was previously disabled when it was in a popover because the calendar was only open while focused on the input.